package/MDAnalysis/core/groups.py

```diff
@@ -3147,7 +3155,8 @@ def select_atoms(self, sel, *othersel, periodic=True, rtol=1e-05,
                                                    periodic=periodic,
                                                    atol=atol, rtol=rtol,
                                                    sorted=sorted,
-                                                   rdkit_kwargs=rdkit_kwargs)
+                                                   rdkit_kwargs=rdkit_kwargs,
```

Comment:
Please add a versionchanged entry for this.

Reply:
not sure what you mean?

Reply:
In the docstring above - add a versionchanged entry